action.yaml

```diff
@@ -0,0 +1,66 @@
+# action.yml
+name: "codacy-analysis-cli"
+description: "Execute Codacy code analysis using your remote Codacy configuration"
+branding:
+  icon: 'check'
+  color: 'gray-dark'
+inputs:
+  project-token:
+    required: true
+  directory:
+    required: false
+    default: "`pwd`"
+  tool-timeout:
+    required: false
+    default: "15minutes"
+  output:
+    required: false
+  format:
+    required: false
+    default: "text"
+  upload:
+    required: false
+    default: false
+  parallel:
+    required: false
+    default: 2
+  max-allowed-issues:
+    required: false
+    default: 0
+  fail-if-incomplete:
+    required: false
+    default: false
+  allow-network:
+    required: false
+    default: false
+  force-file-permissions:
+    required: false
+    default: false
+runs:
+  using: "docker"
+  image: "docker://codacy/codacy-analysis-cli:stable"
+  args:
```

Comment:
The `--tool` option is missing, but I'm unsure of how to support it, due to the lack of conditionals support in GH actions yaml.